<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mode_run</class>
 <widget class="QWidget" name="Mode_run">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Roll Pitch Yaw</string>
   </property>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBoxRoll">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NR</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_48">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBoxPitch">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NR</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_54">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Yaw:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBoxYaw">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NR</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_79">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tilt:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_tilt_1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NR</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_change_mode_run">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode Run</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="QLabel" name="label_61">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VX:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_pidt_vx">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_62">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VY:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBoxVY">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_63">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OMEGA:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_pidt_w">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_control_enable">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
